.mvn/maven.config

```diff
@@ -1 +0,0 @@
--Pbuild-sources-and-javadoc
```

Comment:
Hvorfor slettes denne filen? Har vi annen mekanisme nå for å bygge source- og javadoc-jar?

Reply:
ja, det ligger i action-en.  https://github.com/digipost/action-maven-publish

Reply:
Men hvorfor legges sånt i action-en? Det betyr jo at når jeg bygger et SNAPSHOT selv lokalt, og når action-en gjør det, så får jeg ikke samme output?

Reply:
Hvorfor får du forskjellig output?

Reply:
åja, sånn ja. Så du tenker at det burde være default at det bygges?

Reply:
Fordi når jeg kjører `mvn clean install` så vil vel ikke profilen som inkluderer bygging av sources av javadocs bli aktivert lenger. `.mvn/maven.config` har kommandolinje-argumenter man ønsker skal være med på alle mvn-kjøringer.

Reply:
Hvis jeg har fortstått det riktig så blir profilen `build-sources-and-javadoc` aktivert av action både ved snapshot- og release-bygg. Så da lurer jeg på hvorfor den ikke kan fortsette å være aktivert alltid, konfigurert "the Maven way", i stedet for å spre dette rundt omkring?

Reply:
Da kunne vi vel ha aktivert det i super-parent-pom i stedet? Så slipper vi kaste denne profilen rundt?

Reply:
Det er en mulighet ja. Grunnen til at det har blitt gjort slik er at en parent-pom ikke bør være en sånn one size fits all, men heller har et absolutt minimum man trenger. Jeg har ønsket at prosjekter selv spesifiserer hva som skal inngå i byggene sine. Har sett for mange eksempler på parent-POMs som spec'er opp alt mulig inkludert i bygg, og at man da heller begynner å jobbe seg rundt det de stedene det ikke passer sånn heeeelt 💯%.

Dette trikset her med å tilby en profil gjør at man i parent-POMen tilbyr dette som "valgfri tooling", og man har en enkel måte å aktivere det på fremfor å legge inn XML-config for disse to pluginene der man vil bygge source og javadocs.